sci-libs/primegen/primegen-0.97.recipe

```diff
@@ -43,6 +43,11 @@ BUILD_REQUIRES="
 	haiku_devel
 	"
 
+TEST_PREREQUIRES="
+	cmd:md5sum
+	"
+BUILD_PREREQUIRES += TEST_PREREQUIRES
```

Comment:
I guess you meant
```
BUILD_PREREQUIRES+="$TEST_PREREQUIRES"
```

Reply:
Sorry, I should have properly checked...

Reply:
No problem  ;-)